packages/core/src/types/interfaces/i-cell-data.ts

```diff
@@ -73,6 +73,7 @@ export interface ICellMarks {
     tr?: ICellMarksStyle;
     bl?: ICellMarksStyle;
     br?: ICellMarksStyle;
+    isSkip?: boolean;
```

Comment:
This property appears in 6 different places. Not smelling good. What does it mean?

Reply:
use to skip extension render; each extension manages its own rendering logic

Reply:
create a ticket optimization later and put permission on the top layer to caculate @ybzky